packages/moonstone/CHANGELOG.md

```diff
@@ -16,6 +17,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 - Fonts to correctly use the new font files and updated the international font name from "Moonstone LG Display" to "Moonstone Global"
 - `moonstone/Dropdown` `children` propType so it supports the same format as `ui/Group` (an array of strings or an array of objects with props)
+- `moonstone/FormCheckbox`, `moonstone/Input`, `moonstone/ProgressBar`, `moonstone/RadioItem`, `moonstone/SwitchItem`, and `moonstone/Tooltip` light skin colors.
```

Comment:
This looks like a stray

Reply:
it was in the wrong spot and i moved it when resolving the merge conflict